.github/workflows/cross-browser-test.yml

```diff
@@ -0,0 +1,34 @@
+name: Cross-Browser Test Flow
+
+on:
+  push:
+    branches:
+      - '*'
+      - '*/*'
+      - '!master'
+  pull_request:
+
+jobs:
+  cross-browser-test:
+    name: Cross-Browser (Saucelabs) Tests
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push'
+    env:
```

Comment:
I assume these have already been defined in the settings?

Reply:
Yes, the env variables are in the repo's secrets already.